core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.ts

```diff
@@ -93,6 +98,20 @@ angular.module('oppia').component('contributorDashboardPage', {
           activeTabDetail.customizationOptions.indexOf('language') !== -1);
       };
 
+      ctrl.onChangeTopic = function(topicName: string) {
+        ctrl.topicName = topicName;
+        TranslationTopicService.setActiveTopicName(ctrl.topicName);
+        LocalStorageService.updateLastSelectedTranslationTopicName(
+          ctrl.topicName);
+        $rootScope.$applyAsync();
+      };
+
+      ctrl.showTopicSelector = function() {
+        var activeTabDetail = ctrl.tabsDetails[ctrl.activeTabName];
+        return (
+          activeTabDetail.customizationOptions.indexOf('topic') !== -1);
```

Comment:
Instead of `indexOf()` can't we use [Array.prototype.includes()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)? Here and below.

Reply:
Sure, we can use that, but line 93 of this file https://github.com/oppia/oppia/blob/develop/core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.ts#L93 also uses `indexOf()`. Would it be better to keep them consistent?

Reply:
I think indexOf() is probably fine -- IIRC, includes() has some issues with backward compatibility in some browsers (though I don't know if this still is an issue).